systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/internalClients/KafkaAdminClients.java

```diff
@@ -73,15 +73,15 @@ public void setReplicationFactor(int replicationFactor) {
         this.replicationFactor = replicationFactor <= 0 ? 1 : replicationFactor;
     }
 
-    public String getTopicOperation() {
+    public AdminClientOperations getTopicOperation() {
```

Comment:
Is it _topic operation_ if it returns `AdminClientOperations` which seems more generic? Maybe better name would make it more reusable.

Reply:
Yes, make sense. Good catch.

Reply:
Even five minutes seem to be like awfully lot of waiting.